src/dsp_tools/xmllib/models/root.py

```diff
@@ -94,3 +86,27 @@ def write_file(self, filepath: str | Path) -> None:
                 f"but the following Schema validation error(s) occurred: {err.message}"
             )
             warnings.warn(DspToolsUserWarning(msg))
+
+    def _make_root(self) -> etree._Element:
```

Comment:
This is a duplication of the public `make_root()` above. Is this intentional?

Reply:
No, that is a merge conflict error. Thanks